client/packages/system/src/Stock/Components/StockLineForm.tsx

```diff
@@ -193,6 +196,20 @@ export const StockLineForm = ({
               />
             }
           />
+          <StyledInputRow
+            label={t('label.reason')}
+            Input={
+              <ReasonOptionsSearchInput
+                width={INPUT_WIDTH}
+                type={ReasonOptionNodeType.PositiveInventoryAdjustment}
+                value={draft.reasonOption}
+                onChange={reason => onUpdate({ reasonOption: reason })}
+                reasonOptions={reasonOptions ?? []}
+                loading={loading}
+                disabled={draft?.totalNumberOfPacks === 0}
+              />
+            }
+          />
```

Comment:
Hm, I think reason was originally outside of this main form because its only required when creating new stock line. It shouldn't be there when just editing stock line details:

![Screenshot 2025-07-08 at 10 46 52 AM](https://github.com/user-attachments/assets/6f4739ed-3cfa-4002-88eb-4ca7c63612df)

I think either a `showReason` prop, or pass in "extraFields" to render?